packages/components/pie-modal/src/index.ts

```diff
@@ -387,8 +387,8 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
         <dialog
             id="dialog"
             class="c-modal"
-            size="${size}"
-            position="${position}"
+            size="${size || 'medium'}"
+            position="${position || 'center'}"
```

Comment:
Since we want to make sure `size` and `position` are always set, the default values are passed again, instead of using `ifDefined` which would omit the attribute in case the user passes something like "falsey" value.

What bothers me here is repeating the default value, since we already use them in the class fields. I believe this can be error-prone since we define the same values in different places using plain strings.

I wonder if it wouldn't be better if we had a single immutable object that stores the component default values, so we could use its reference wherever needed, saving us from potential regressions.

Reply:
If repeating is indeed unavoidable then I prefer this approach

Reply:
Agreed - the default object approach sounds like a great idea @fernandofranca!